universitat-bremen-institut-fur-politikwissenschaft.csl

```diff
@@ -233,7 +267,11 @@
           <text macro="published-date"/>
           <choose>
             <if match="any" variable="page">
-              <text macro="pages"/>
+              <choose>
+                <if match="none" variable="DOI">
```

Comment:
no, this isn't right. The instructions say to only do this for e-only publication (Reine E-Journals). You're getting rid of page ranges for all journal articles with DOIs, which is pretty much any journal article published in the last 5 years (probably longer).